app/jobs/registration_reminder_job.rb

```diff
@@ -2,10 +2,10 @@ class RegistrationReminderJob < ApplicationJob
   queue_as :default
 
   def perform(*args)
-    Event.where('start_time > ? and start_time < ?', Time.now, Time.now.tomorrow).each do |e|
-      EventMailer.reminder e
+    Event.where('start_time > ? and start_time < ?', Time.zone.now + 1.days, Time.zone.now + 2.days).each do |e|
+      EventMailer.delay.reminder(e)
```

Comment:
This mailer looks like it sends an email to all of the attendees.

Reply:
Also, `EventMailer.reminder(e).deliver_later` will be a slightly better since it uses whatever you have set up for ActiveJob, while `.delay` is specific to `delayed_job`.

Reply:
I'll implement.